fix(support link): auto-fill organization when on org page

charislam opened the pull request:
If user clicks the Support Link from an org page (not a project page), pass along the organization slug in the query params to auto-fill the Support Form.

Resolves FE-1611

charislam committed:
fix(support link): auto-fill organization when on org page

If user clicks the Support Link from an org page (not a project page),
pass along the organization slug in the query params to auto-fill the
Support Form.

awaseem reviewed (approved):
tested in preview!, LGTM

charislam committed:
fix(date picker test)

The new UI for range picking is a little different, and requires
double-clicking to start a new range.
